awswrangler/athena/_read.py

```diff
@@ -586,8 +586,8 @@ def get_query_results(
         Recommended for memory restricted environments.
     chunksize : Union[int, bool], optional
         If passed will split the data in a Iterable of DataFrames (Memory friendly).
-        If `True` wrangler will iterate on the data by files in the most efficient way without guarantee of chunksize.
-        If an `INTEGER` is passed Wrangler will iterate on the data by number of rows igual the received INTEGER.
+        If `True` awswrangler iterates on the data by files in the most efficient way without guarantee of chunksize.
```

Comment:
Do we want to remove references to `wrangler` like this one from the code? I understand the PyPi package will still have the same name, so it's not a huge deal. But perhaps we could avoid naming the library in these instances altogether, just to reduce confusion. For example:
```
If `True`, the function iterates on the data by files in the most efficient way without guarantee of chunksize.
```

Reply:
@LeonLuttenberger Good catch and I agree as this will be on our docs page which I think we want to mention wrangler as little as possible.